src/main/scala/org/constellation/lb/Manager.scala

```diff
@@ -231,3 +234,23 @@ class Manager(init: NonEmptyList[Addr], config: LoadbalancerConfig)(
       )
   }
 }
+
+object Manager {
+
+  val logger: Logger[IO] = Slf4jLogger.getLogger[IO]
```

Comment:
get logger for class

Reply:
I've actually, got rid of the logger here, and split the code into two functions, leaving the logging in the main calling method